README.md

```diff
@@ -18,18 +22,18 @@ Twilio and Java Servlets.
 1. Create the database.
 
     ```bash
-    $ createdb airtng-servlets
+    $ createdb ivr-recording
 
     ```
   _The application uses PostgreSQL as the persistence layer. If you
   don't have it already, you should install it. The easiest way is by
   using [Postgres.app](http://postgresapp.com/)._
 
-1. Edit the sample configuration file `.environment` to match your database       configuration:
+1. Edit the sample configuration file `.environment` to match your database configuration:
```

Comment:
**Not for this one**, but I'm thinking about using always `.env.example` as the canonical way to group the environment variables. Of course this approach will includes adding the `.env` to the _.gitignore_.

What do you think?